glean/src/core/metrics/types/text.ts

```diff
@@ -0,0 +1,89 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import type { CommonMetricData } from "../index.js";
+import { isString, truncateStringAtBoundaryWithError } from "../../utils.js";
+import { MetricType } from "../index.js";
+import { Context } from "../../context.js";
+import { Metric } from "../metric.js";
+
+// The maximum length for text: 200 kilobytes.
```

Comment:
```suggestion
// The maximum length for text: 200 characters.
```

If you want to check by kylobites then the string needs to be encoded before the check. We use a function for that `strToU8`, but we have been checking character length in URL and String metric types until now.

Reply:
hm, fwiw the jsonschema defines the string length as the number of its characters according to the JSON rfc, which defines characters as unicode characters.
So ... uhm ... yeah, lets stick with characters.